fbgemm_gpu/CMakeLists.txt

```diff
@@ -147,14 +171,30 @@ set(codegen_dependencies
     ${CMAKE_CURRENT_SOURCE_DIR}/include/fbgemm_gpu/sparse_ops_utils.h
 )
 
-add_custom_command(
-  OUTPUT ${gen_cpu_source_files} ${gen_gpu_source_files}
-         ${gen_gpu_host_source_files} ${gen_python_files}
-  COMMAND
+if(USE_ROCM)
+	execute_process(
+    COMMAND
     "${PYTHON_EXECUTABLE}"
     "${CMAKE_CODEGEN_DIR}/embedding_backward_code_generator.py"
     "--opensource"
-  DEPENDS "${codegen_dependencies}")
+		DEPENDS "${codegen_dependencies}")
+
+  set(header_include_dir
+      ${CMAKE_CURRENT_SOURCE_DIR}/include
+      ${CMAKE_CURRENT_SOURCE_DIR}/src
+      ${CMAKE_CURRENT_SOURCE_DIR})
+
+  hipify(CUDA_SOURCE_DIR ${PROJECT_SOURCE_DIR} HEADER_INCLUDE_DIR ${header_include_dir})
+else()
+	add_custom_command(
```

Comment:
Ditto. Please follow the original whitespace formatting (in this case, tab -> whitespace)